### DISCUSSION TITLE
An .apk version for use in VR

### DISCUSSION THREAD

**Discussion by metaversodazoeira** [baffler/Transparent-Twitch-Chat-Overlay#277] [Ideas]
I would like to be able to use this application in my VR, I am a content creator and there is no solution to read the chat natively.

**Comment by IAmAProfessionalGamer** [comment]
Gunrun may be more suited for what you are doing. Here's a link to their page, hope this helps: https://www.gunrun.tv/vr-streaming-for-vr-chat

**Reply by metaversodazoeira** [reply]
Thanks, this does not resolve the issue of viewing the chat in a standalone broadcast. It would only work on PCVR games.
